src/main/java/org/gitlab/api/Pagination.java

```diff
@@ -10,21 +10,23 @@
     public static final int MAX_ITEMS_PER_PAGE = 100;
     private final Query paginationQuery = new Query();
 
-    public void setPage(int page) {
+    public Pagination setPage(int page) {
         try {
             paginationQuery.append(PARAM_PAGE, String.valueOf(page));
         } catch (UnsupportedEncodingException ignored) {
         }
+        return this;
     }
 
-    public void setPerPage(int perPage) {
+    public Pagination setPerPage(int perPage) {
```

Comment:
let me know if you consider this a breaking change, could introduce new fluent API with new methods eg. public Pagination withPerPage(int perPage)..